src/vl-rich-data-table.js

```diff
@@ -19,6 +19,7 @@ import {VlRichDataSorter} from "./vl-rich-data-sorter.js";
  * @property {boolean} data-vl-multisort - Laat de gebruiker sorteren op meer dan 1 kolom.
  *
  * @slot filter - slot dat de velden bevat waarop gefilterd wordt. De formData van de search filter worden via een change event doorgegeven bij een wijziging.
+ * @property {boolean} data-vl-filter-closable - Attribuut dat de filter sluitbaar maakt en een knop getoond wordt om de filter te tonen en terug te verbergen.
```

Comment:
Is het eigenlijk ook niet mogelijk om dit op het slot mee te geven?
Dan wordt de api van de table zelf niet zo bloated (want ik kan me voorstellen dat er wel meer van zo'n opties komen).

Reply:
Lijkt me moeilijker om de rendering goed te krijgen. Ik denk dat het aantal uitbreidingen wel zal meevallen.